pkg/controller/discovery/web/pod.go

```diff
@@ -194,7 +194,7 @@ func (h *LogHandler) getLog(ctx *gin.Context, pod *model.Pod) {
 		return
 	}
 	request := podClient.GetLogs(pod.Name, options)
-	stream, err := request.Stream()
+	stream, err := request.Stream(ctx)
```

Comment:
Is this the right kind of context? passing a gin context to a request.Stream?

Reply:
@djwhatle I'm actually not 100% sure. I know we need a context now. Would Context.TODO() be more appropriate? WIth most of the updated changes, things that previously didn't require a context that now do we've been using Context.TODO() if we didn't already have a context, but if there was a current one I tried to use it, but if this particular context makes no sense here, then we should probably use TODO() instead.

Reply:
I can't think of what request.Stream would do with a gin ctx, maybe passing it would mean support for cancellation of the gin request cascading through to the stream call.